code/game/gamemodes/blob/blobs/shield.dm

```diff
@@ -48,7 +48,7 @@
 		P.firer = src //so people who fired the lasers are not immune to them when it reflects
 		visible_message("<span class='warning'>[P] reflects off [src]!</span>")
 		return -1// complete projectile permutation
-	else if(P.is_reflectable && P.legacy) //to stop legacy projectile exploits
+	else if(P.is_reflectable) //to stop legacy projectile exploits

```

Comment:
This will literally break shield blobs. Take out the whole else if block.

Reply:
I can't believe I missed this